app/workers/queue_listener.rb

```diff
@@ -6,22 +6,18 @@ class QueueListener
   # shoryuken_options queue: 'post_test', auto_delete: true, body_parser: :json
 
   def perform(sqs_message, params)
-    params = params.symbolize_keys
```

Comment:
Did you test this with the SQS queue? It converts the params that it posts entirely to strings, which is why we were symbolizing the keys. It looks like this might break page creation for champaign.